Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfImage.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// Image data used to create a texture. Image can be referenced by URI or
+    /// `bufferView` index. `mimeType` is required in the latter case.
+    /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/image.schema.json
+    /// </summary>
+    [Serializable]
+    public class GltfImage : GltfChildOfRootProperty
+    {
+        #region Serialized Fields
+
+        /// <summary>
+        /// The uri of the image.  Relative paths are relative to the .gltf file.
+        /// Instead of referencing an external file, the uri can also be a data-uri.
+        /// The image format must be jpg, png, bmp, or gif.
+        /// </summary>
+        public string uri;
```

Comment:
The MRTK coding convention captializes public fields.

Reply:
I'm very well aware of the coding conventions. I wrote them.

We do not when they're supposed to be serialized by the json serializer.

Reply:
Synced offline: onboard with this naming based on khronos schema declarations